utils.py

```diff
@@ -982,8 +982,10 @@ def get_launch_count(activity):
 
 
 def get_colors():
-    client = GConf.Client.get_default()
-    return XoColor(client.get_string('/desktop/sugar/user/color'))
+    settings = Gio.Settings('org.sugarlabs.user')
+    path = '/desktop/sugar/user/color'
```

Comment:
`GConf` client used the path to get the string, with `Gio` uses existing schemas hence you don't need path.